### PR TITLE
#80 Design admin page

### PR DESCRIPTION
**Pull request by semyonov-m-y**

### CODE REVIEW

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Articles</td> 
+    				<td>25th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Ipsum Lorem Dolor Sit Amet</td> 
+    				<td>Freebies</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Super Hans</td> 
+    				<td>20th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Dolor Lorem Amet</td> 
+    				<td>Alan Johnson</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 3]
```diff
@@ -0,0 +1,329 @@
+<!doctype html>
+<html lang="en">
+
+<head>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MAJOR Add a 'favicon' declaration in this 'header' tag. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Freebies</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr>
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Sit Amet Dolor Ipsum</td> 
+    				<td>Tutorials</td> 
+    				<td>20th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Alan Johnson</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Dolor Lorem Amet</td> 
+    				<td>Dobby</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+		<header><h3 class="tabs_involved">Content Manager</h3>
+		<ul class="tabs">
+   			<li><a href="#tab1">Posts</a></li>
+    		<li><a href="#tab2">Comments</a></li>
+		</ul>
+		</header>
+
+		<div class="tab_container">
+			<div id="tab1" class="tab_content">
+			<table class="tablesorter" cellspacing="0">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MAJOR Remove this deprecated "cellspacing" attribute. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+    				<th>Actions</th> 
+				</tr> 
+			</thead> 
+			<tbody> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Lorem Ipsum Dolor Sit Amet</td> 
+    				<td>Mark Corrigan</td> 
+    				<td>25th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+				</form>
+			</footer>
+		</article>
+		
+		<div class="clear"></div>
+		
+		<article class="module width_full">
+			<header><h3>Post New Article</h3></header>
+				<div class="module_content">
+						<fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Tutorials</td> 
+    				<td>20th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Dolor Lorem Amet</td> 
+    				<td>Articles</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+		<div class="clear"></div>
+		
+		<article class="module width_full">
+			<header><h3>Post New Article</h3></header>
+				<div class="module_content">
+						<fieldset>
+							<label>Post Title</label>
+							<input type="text">
+						</fieldset>
+						<fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 11]
```diff
@@ -0,0 +1,329 @@
...
+    $(function(){
+        $('.column').equalHeight();
+    });
+</script>
+
+</head>
+
+<body>
+	<header id="header">
+		<hgroup>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MAJOR Remove this deprecated "hgroup" element. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AUnsupportedTagsInHtml5Check)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 12]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Mark Corrigan</td> 
+    				<td>25th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Ipsum Lorem Dolor Sit Amet</td> 
+    				<td>Jeremy Usbourne</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 13]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Articles</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr>
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Dolor Lorem Amet</td> 
+    				<td>Articles</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 14]
```diff
@@ -0,0 +1,329 @@
...
+				<div class="module_content">
+						<fieldset>
+							<label>Post Title</label>
+							<input type="text">
+						</fieldset>
+						<fieldset>
+							<label>Content</label>
+							<textarea rows="12"></textarea>
+						</fieldset>
+						<fieldset style="width:48%; float:left; margin-right: 3%;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 15]
```diff
@@ -0,0 +1,329 @@
...
+    				<th>Actions</th> 
+				</tr> 
+			</thead> 
+			<tbody> 
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Lorem Ipsum Dolor Sit Amet</td> 
+    				<td>Articles</td> 
+    				<td>25th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 16]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Jeremy Usbourne</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr>
+				<tr> 
+   					<td><input type="checkbox"></td> 
+    				<td>Sit Amet Dolor Ipsum</td> 
+    				<td>Super Hans</td> 
+    				<td>20th May 2017</td> 
+    				<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)
MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 17]
```diff
@@ -0,0 +1,329 @@
...
+    				<td>Articles</td> 
+    				<td>26th May 2017</td> 
+   				 	<td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image" src="adminimg/icn_trash.png" title="Trash"></td> 
+				</tr>  
+			</tbody> 
+			</table>
+			</div>
+			
+			<div id="tab2" class="tab_content">
+			<table class="tablesorter" cellspacing="0">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MAJOR Remove this deprecated "cellspacing" attribute. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 18]
```diff
@@ -0,0 +1,329 @@
...
+							<textarea rows="12"></textarea>
+						</fieldset>
+						<fieldset style="width:48%; float:left; margin-right: 3%;"> 
+							<select style="width:92%;">
+								<option>Articles</option>
+								<option>Tutorials</option>
+								<option>Freebies</option>
+							</select>
+						</fieldset>
+						<fieldset style="width:48%; float:left;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/adminjs/jquery.equalHeight.js [hunk 19]
```diff
@@ -0,0 +1,20 @@
+// make sure the $ is pointing to JQuery and not some other library
+    (function($){
+        // add a new method to JQuery
+
+        $.fn.equalHeight = function() {
+           // find the tallest height in the collection
+           // that was passed in (.column)
+            tallest = 0;
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/adminjs/jquery.equalHeight.js]: BLOCKER Add the "let", "const" or "var" keyword to this declaration of "tallest" to make it explicit. [rule](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)

File: src/main/resources/static/admin/adminjs/jquery.equalHeight.js [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+            this.each(function(){
+                thisHeight = $(this).height();
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/adminjs/jquery.equalHeight.js]: BLOCKER Add the "let", "const" or "var" keyword to this declaration of "thisHeight" to make it explicit. [rule](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 30 issues
* ![BLOCKER][BLOCKER] 2 blocker
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 24 minor

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 21]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Articles</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 22]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Ipsum Lorem Dolor Sit Amet</td>
+                        <td>Freebies</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 23]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Dobby</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 24]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Ipsum Lorem Dolor Sit Amet</td>
+                        <td>Jeremy Usbourne</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 25]
```diff
@@ -0,0 +1,369 @@
...
+            </form>
+        </footer>
+    </article>
+
+    <div class="clear"></div>
+
+    <article class="module width_full">
+        <header><h3>Post New Article</h3></header>
+        <div class="module_content">
+            <fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 26]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Sit Amet Dolor Ipsum</td>
+                        <td>Tutorials</td>
+                        <td>20th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 27]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Sit Amet Dolor Ipsum</td>
+                        <td>Super Hans</td>
+                        <td>20th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 28]
```diff
@@ -0,0 +1,369 @@
...
+    <div class="clear"></div>
+
+    <article class="module width_full">
+        <header><h3>Post New Article</h3></header>
+        <div class="module_content">
+            <fieldset>
+                <label>Post Title</label>
+                <input type="text">
+            </fieldset>
+            <fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 29]
```diff
@@ -0,0 +1,369 @@
...
+                <label>Content</label>
+                <textarea rows="12"></textarea>
+            </fieldset>
+            <fieldset style="width:48%; float:left; margin-right: 3%;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 30]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Articles</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 31]
```diff
@@ -0,0 +1,369 @@
...
+                        <th>Actions</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Lorem Ipsum Dolor Sit Amet</td>
+                        <td>Articles</td>
+                        <td>25th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 32]
```diff
@@ -0,0 +1,369 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Alan Johnson</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 33]
```diff
@@ -0,0 +1,369 @@
...
+                        <th>Actions</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Lorem Ipsum Dolor Sit Amet</td>
+                        <td>Mark Corrigan</td>
+                        <td>25th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 34]
```diff
@@ -0,0 +1,369 @@
...
+                <textarea rows="12"></textarea>
+            </fieldset>
+            <fieldset style="width:48%; float:left; margin-right: 3%;">
+                <select style="width:92%;">
+                    <option>Articles</option>
+                    <option>Tutorials</option>
+                    <option>Freebies</option>
+                </select>
+            </fieldset>
+            <fieldset style="width:48%; float:left;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 14 issues
* ![MINOR][MINOR] 14 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/resources/static/admin/AdminPanelPage.html [hunk 35]
```diff
@@ -0,0 +1,344 @@
...
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Articles</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```
Comment (Sheptytskyid) [inline src/main/resources/static/admin/AdminPanelPage.html]: MINOR Add an "alt" attribute to this image. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

**Comment by GetFire** [comment]
• папка админ должна лежать в /static/views/
•все .css вынеси в соответствующую папку
• уверен все .png можно выбросить и заменить на иконки из библиотеки fonts-awesome. Тем более, что она уже есть у нас в проекте
•зачем нам сейчас в админке статистика? тем более такая(просто png). Я считаю, что сейчас в админке  нужно реализовать полностью рабочий CRUD функционал,  при чем для всех сущностей. 
• очень сложно могу себе представить как подключить это все и заставить работать.
• это хороший **шаблон** для dashboadr. Но нам нужна более легкая версия, чтобы подключить ее и работать
• почему ты не подключил это к ангуляру ?

**Comment by semyonov-m-y** [comment]
1. Сделал в расчёте на рост проекта. Не знаю нужно ли чтобы сейчас всё работало. 
2. Смотрю на разные проекты, css и js файлы раскиданы по разным папкам - не стоит всё пихать в одну, тем более что мы не используем в админке общие для других страниц файлов.
3. По .png подумаю
4. После утверждения окончательного вида админ панели, подключу к Ангуляру

File: src/main/resources/static/views/admin/AdminPanelPage.html [hunk 36]
```diff
@@ -0,0 +1,284 @@
...
+                    <textarea rows="12"></textarea>
+                </fieldset>
+                <fieldset style="width:48%; float:left; margin-right: 3%;">
+                    <select style="width:92%;">
+                        <option>Articles</option>
+                        <option>Tutorials</option>
+                        <option>Freebies</option>
+                    </select>
+                </fieldset>
+                <fieldset style="width:48%; float:left;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/views/admin/AdminPanelPage.html:240]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/views/admin/AdminPanelPage.html [hunk 37]
```diff
@@ -0,0 +1,284 @@
...
+                </form>
+            </footer>
+        </article>
+
+        <div class="clear"></div>
+
+        <article class="module width_full">
+            <header><h3>Post New Article</h3></header>
+            <div class="module_content">
+                <fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/views/admin/AdminPanelPage.html:225]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/views/admin/AdminPanelPage.html [hunk 38]
```diff
@@ -0,0 +1,284 @@
...
+                    <label>Post Title</label>
+                    <input type="text">
+                </fieldset>
+                <fieldset>
```
Comment (Sheptytskyid) [inline src/main/resources/static/views/admin/AdminPanelPage.html:229]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

File: src/main/resources/static/views/admin/AdminPanelPage.html [hunk 39]
```diff
@@ -0,0 +1,284 @@
...
+                    <label>Content</label>
+                    <textarea rows="12"></textarea>
+                </fieldset>
+                <fieldset style="width:48%; float:left; margin-right: 3%;">
```
Comment (Sheptytskyid) [inline src/main/resources/static/views/admin/AdminPanelPage.html:233]: MINOR Add a <legend> tag to this fieldset. [rule](https://sonarcloud.io/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 4 issues
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by AlexPoliteRudenko** [comment]
У мене чомусь не підтягнулися стилі. Сильно не перевіряв. Кидаю скрін 
image

File: src/main/resources/static/css/ie.css [hunk 40]
```diff
@@ -0,0 +1,43 @@
+.quick_search {
+text-align: center;
+padding: 14px 0 0px 0;
+}
+
+.quick_search input[type=text] {
+text-align: left;
```
Comment (zozich) [inline src/main/resources/static/css/ie.css:7]: Where are your paddings?

File: src/main/resources/static/js/vendor/event.js [hunk 41]
```diff
@@ -0,0 +1,25 @@
+	$(document).ready(function() 
+    	{
```
Comment (zozich) [inline src/main/resources/static/js/vendor/event.js:2]: Where did you see this kind of formatting?

**Review by zozich — APPROVED** [review]
Ok...

**Pull request merged**